R/tidyomics.R

```diff
@@ -16,3 +16,22 @@ release_bullets <- function() {
     '`tidyomics_dependency_dissuade()`'
   )
 }
+
+#' tidyomics
+#' 
+#' @description A helper package to install and attach the core packages of the tidyomics ecosystem.
+#' @importFrom BiocManager install
+#' @export 
+#' @return No value is returned
+#' @examples
+#' \dontrun{
+#' # Install the core packages of the tidyomics ecosystem
+#' BiocManager::install("tidyomics")
+#' 
+#' # Attach the core packages of the tidyomics ecosystem
+#' library(tidyomics)
+#' }
+tidyomics <-
+  function() {
+    return(NULL)
+  }
```

Comment:
I think this should not be here. A function that returns null is confusing.

Reply:
This addition was requested by the Bioconductor reviewer to help a user who enters `?tidyomics` when trying to understand how to use the package.